reconcile/utils/git.py

```diff
@@ -30,5 +30,8 @@ def checkout(commit, wd):
 def is_file_in_git_repo(file_path):
     real_path = os.path.realpath(file_path)
     dir_path = os.path.dirname(real_path)
-    git_path = os.path.join(dir_path, '.git')
-    return os.path.exists(git_path)
+    with open(os.devnull, 'w') as dev_null:
+        proc = Popen(['git', 'git rev-parse', '--is-inside-work-tree'],
```

Comment:
the first parameter to the git command should be only `rev-parse` not `git rev-parse`

Reply:
can't a guy make typos in peace? :wink: 
fixed in https://github.com/app-sre/qontract-reconcile/pull/2066/commits/825c961a1ce9e3afa76f8807032793424012cd12

Reply:
No 😛